code/modules/mob/abstract/new_player/sprite_accessories.dm

```diff
@@ -4181,6 +4305,12 @@ Follow by example and make good judgement based on length which list to include
 			icon_state = "hand_panel_l"
 			body_parts = list(BP_L_HAND)
 
+	autopsy
+		name = "Autopsy Scar"
+		icon_state = "autopsyscar"
+		body_parts = list(BP_CHEST)
+		species_allowed = list(/datum/species/human, /datum/species/human/offworlder, /datum/species/machine/shell, /datum/species/diona, /datum/species/diona/coeu, /datum/species/tajaran, /datum/species/tajaran/zhan_khazan, /datum/species/tajaran/m_sai, /datum/species/unathi)
```

Comment:
```suggestion
		species_allowed = list(/datum/species/human, /datum/species/human/offworlder, /datum/species/diona, /datum/species/diona/coeu, /datum/species/tajaran, /datum/species/tajaran/zhan_khazan, /datum/species/tajaran/m_sai, /datum/species/unathi)
```

Reply:
Yeah, I forgot to remove them from the copy paste. Will commit.